config/polygon.json

```
@@ -0,0 +1,27 @@
+{
+    "network": "matic",
+    "erc721": [
+        {
+            "name": "AavegotchiDiamond",
+            "address": "0x86935f11c86623dec8a25696e1c19a8659cbf95d",
+            "startBlock": 11516320
+        },
+        {
+            "name": "RealmDiamond",
```

Let's always use the name of the game in the name of the NFT
```suggestion
            "name": "AavegotchiParcel",
```